### PR TITLE
fix: board resizing on right and bottom

### PR DESCRIPTION
**Pull request by 3ddelano**
Fixes the issue where board resizing didnt work and gave a padding on tiles and the entire board on bottom and right.

### CODE REVIEW

**Review by bend-n — APPROVED** [review]
how does this work? and why?

**Comment by 3ddelano** [comment]
> how does this work? and why?

The issue is with the colored background of the board scene, this might not be a permanent fix but it works. I suspect that the board scene isnt expanding well when its in the game scene.

File: ui/board/Game.gd [hunk 1]
```diff
@@ -11,6 +12,9 @@ onready var panels := [
 func _ready() -> void:
 	PacketHandler.connect("info_recieved", self, "_spectate_info" if Globals.spectating else "_on_info")
 
+	sidebar.visible = false
```
Comment (3ddelano) [inline ui/board/Game.gd:15]: Line 15 and line 16 are the fix for the resizing.

**Comment by 3ddelano** [comment]
Btw only line 15 and 16 in Game.gd is the fix

**Pull request merged**